conf/st2.package.conf

```diff
@@ -52,7 +52,7 @@ api_url =
 base_path = /opt/stackstorm
 
 [syslog]
-host = localhost
+host = 127.0.0.1
```

Comment:
These are all configurable values. If something does not bind to 127.0.0.1 then change config. Since this is install specific not much we can do here - this is being done by AIO.